apps/studio/tests/integration/helpers/seed/index.ts

```diff
@@ -0,0 +1,264 @@
+import { db, jsonb } from "~server/db"
+import { nanoid } from "nanoid"
+
+import type { ResourceState, ResourceType } from "~server/db"
+
+export const setupSite = async (siteId?: number, fetch?: boolean) => {
+  if (siteId !== undefined && fetch) {
+    return db.transaction().execute(async (tx) => {
+      const site = await tx
```

Comment:
these 3 ops seems to be fetching the existing site, nav, footer configs. Why are they within the setupSite function?

Reply:
a site is always the triple <site, footer, navbar>. so if fetching we fetch lo; if not we create